independent-publisher-connectors/Shopify Admin Content/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2895 @@
+    }
+  },
+  "host": "thatapiguy.myshopify.com",
+  "basePath": "/admin/api/2021-04/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/blogs/{blog_id}/articles.json": {
+      "get": {
+        "description": "Retrieves a list of all articles from a certain blog",
+        "summary": "Get a list of all articles of a blog",
+        "tags": [
+          "Article"
+        ],
+        "operationId": "ListBlogArticles",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "blog_id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The ID of the blog containing the articles.",
+            "x-ms-summary": "Blog ID",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "articles": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "id"
+                      },
+                      "title": {
+                        "type": "string",
+                        "description": "title"
+                      },
+                      "created_at": {
+                        "type": "string",
+                        "description": "created_at"
+                      },
+                      "body_html": {
+                        "type": "string",
+                        "description": "body_html"
+                      },
+                      "blog_id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "blog_id"
+                      },
+                      "author": {
+                        "type": "string",
+                        "description": "author"
+                      },
+                      "user_id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "user_id"
+                      },
+                      "published_at": {
+                        "type": "string",
+                        "description": "published_at"
```

Comment:
@thatapiguy Can it be `Published at" ?

Documentation - [Description of properties](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

Applies to all the similar description values in swagger

Reply:
Changed to Published at and changed other similar description values as well